docs/feature_unicode.md

```diff
@@ -135,11 +142,12 @@ You can find the default implementations of these functions in [`process_unicode
 
 #### Input Key Configuration
 
-Additionally, you can customize the keys used to trigger the unicode input for macOS and WinCompose by adding defines to your `config.h`
+You can customize the keys used to trigger Unicode input for Mac, Linux and WinCompose by adding defines to your `config.h`. For example:
 
 ```c
-#define UNICODE_OSX_KEY  KC_LALT
-#define UNICODE_WINC_KEY KC_RALT
+#define UNICODE_KEY_OSX  KC_RALT
+#define UNICODE_KEY_LNX  LCTL(LSFT(KC_E))
+#define UNICODE_KEY_WINC KC_RGUI
```

Comment:
IMO this section should be tabulated. Compare the [Backlight docs](https://docs.qmk.fm/#/feature_backlight?id=configuration).

Reply:
Done.